app/(routes)/organizers/page.tsx

```diff
@@ -138,29 +128,56 @@ const team = [
   },
   {
     name: "Claudio Olmedo",
-    role: "Volunteer",
+    role: "Organizer",
     linkedinUrl: "https://www.linkedin.com/in/claudioolmedo-com/",
     // teamLead: 'Team Lead:',
     team: '',
     imageUrl:
       "https://media.licdn.com/dms/image/D4E03AQE6q5fDJj5zcA/profile-displayphoto-shrink_200_200/0/1685118707659?e=1695254400&v=beta&t=MTVN3lH51OFLr27bpGGep401OW3RffbA8vNeANqZGZY"
   },
+  {
+    name: "Benjamin Chen",
+    role: "Organizer",
+    linkedinUrl: "https://www.linkedin.com/in/benjamin-chen-a7a12750/",
+    // teamLead: 'Team Lead:',
+    team: '',
+    imageUrl:
+      "https://media.licdn.com/dms/image/C4D03AQEdG-JySQY0FQ/profile-displayphoto-shrink_800_800/0/1593446721248?e=1699488000&v=beta&t=eXADY-r2VoqqJL0cEA3KwsEj7-k2GMYATsEfDbsrhGI"
+  },
+  {
+    name: "Anirban Dutta",
+    role: "Organizer",
+    linkedinUrl: "https://www.linkedin.com/in/anirbandutta7/",
+    // teamLead: 'Team Lead:',
+    team: '',
+    imageUrl:
+      "https://media.licdn.com/dms/image/D4E03AQEeqg9cDqm9gg/profile-displayphoto-shrink_800_800/0/1667252731966?e=1699488000&v=beta&t=NWuVkDGLA1d81WnIQabtPBTrRPiMQoJ7rWPJ_qUiPMU"
+  },
+  // {
+  //   name: "Anmol Vijay Bhatia",
+  //   role: "Organizer",
+  //   linkedinUrl: "https://www.linkedin.com/in/anmol-vb/",
+  //   // teamLead: 'Team Lead:',
+  //   team: '',
+  //   imageUrl:
+  //     "https://media.licdn.com/dms/image/C5603AQGIMDZ25uvPqQ/profile-displayphoto-shrink_800_800/0/1658655544904?e=1699488000&v=beta&t=YvEoffE9F8jPxXDWgi8rhgLGH9j88cunH_b2ud8Jc0g"
+  // },
 
 
 
 ];
```

Comment:
This file is getting pretty lengthy. This hard coded data could be pulled into a separate file...